File: 0058-length-of-last-word/0058-length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int p = size(s)-1,count=0;
        while(p>=0)
        {
            if(s[p]!=' ') ++count;
            else if(count>0) break;
            --p;
        }
        return count;
    }
};

File: 0058-length-of-last-word/0058-length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
        int l=0;
        String str=s.trim();
        for(int i=str.length()-1;i>=0;i--){
            if(str.charAt(i)!=' '){
                l++;
            }
            else{
                break;
            }
        }
        return l;
    }
}